presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -39,6 +42,7 @@
 @NotThreadSafe
 public class MethodDefinition
 {
+    private static final int MAX_ALLOWED_ESTIMATED_SIZE = 128 * 1024;
```

Comment:
should this be configurable or is this a hard limit no matter your jvm configuration?

Reply:
This is a 2x64K which is the JVM hard limit.